go/teams/handler.go

```diff
@@ -363,11 +363,25 @@ func handleSeitanSingle(ctx context.Context, g *libkb.GlobalContext, team *Team,
 		return err
 	}
 
-	ikey, err := peikey.DecryptIKey(ctx, team)
+	ikeyAndLabel, err := peikey.DecryptIKeyAndLabel(ctx, team)
 	if err != nil {
 		return err
 	}
 
+	var ikey SeitanIKey
+
+	version, err := ikeyAndLabel.V()
+	if err != nil {
+		return fmt.Errorf("While parsing IKeyAndLabel: %s\n", err)
+	}
+
+	switch version {
+	case keybase1.SeitanIKeyAndLabelVersion_V1:
+		ikey = SeitanIKey(ikeyAndLabel.V1().I)
+	default:
+		return fmt.Errorf("Unknown IKeyAndLabel version: %v\n", version)
```

Comment:
Client will cancel the invite which is probably not what we want, because it probably means that client is outdated and someone else should process this message. On the other hand, someone can post corrupted invitation and if clients don't cancel it, we again end up in a situation where rekeyd retries it infinitely.

Reply:
Can CLKR clients cancel broken invites? That makes the most sense.